studips/src/component/MainThread.js

```diff
@@ -208,6 +244,38 @@ class MainThread extends React.Component {
 			setTimeout(this.getThread(), 1000),
 		);
 	}
+	handleSubmitPrivateMessage(e) {
+		e.preventDefault()
+		const token = this.props.token
+		const axiosConfig = {
+        	headers: {
+				'Content-Type': 'application/json',
+				'Authorization': 'Bearer ' + token
+    		}
+    	}
+		const contactId = this.state.recipientId
+		const newPvMessage = {
+			sender_id: this.state.userData.id,
+			recipient_id: this.state.recipientId,
+			content: this.state.newPv.content
+		}
+		axios
+			.post('http://localhost:8000/conversation', newPvMessage, axiosConfig)	
+			.then(res => console.log(res))
+			.catch(err => console.log(err))
+			.then(setTimeout(() => this.getConversationAfterPv(contactId), 150))
+	}
+	handleChangeNewPvMess(event) {
+		const propertyName = event.target.name;
+		const newPvMess = this.state.newPv;
```

Comment:
Be careful : you do not make a copy of object this.state.newPv here. newPvMess & this.state.newPv both refers to the same & unique object at that point -- both refers to the same value. If you want newPvMess to be a copy of this.state.newPv, use spread operator

Reply:
Fixed :-)